gasReporterOutput.json

```diff
@@ -0,0 +1,57 @@
+·----------------------------------------------------------------------|---------------------------|-------------|-----------------------------·
```

Comment:
Don't commit this file. Maybe add to gitignore

Reply:
removed